indicators/NO_GJEN/R/NO_GJEN_004.qmd

```diff
@@ -0,0 +1,1467 @@
+
+```{r enc-workflow, fig.cap="Schematic illustration of how the encroachment indicator is calculated. The lower bound for poor condition is set by LiDAR heights from mature forest surrounding the population polygon. The upper bound for good condition is set by regional median LiDAR heights within NiN polygons in the same bioclimatic-elevation zone as the population polygon.", out.width='70%', echo=FALSE}
+knitr::include_graphics("../img/gjengroing_schematic.jpg")
+```
+
+The NiN polygons including "Våtmark", "Naturlig åpne områder under skoggrensa" and "Semi-naturlig mark" with good ecological condition are used to define a reference gjengroing state. We use the aggregated "Tilstand" variable assigned to each NiN polygon. The 50th percentile of LiDAR-derived vegetation heights within these polygons is used to define the upper limit (ie. 1) of the scaled indicator value. We cannot define local reference values based on proximity, because the NiN polygons are spatially biased and not close to all population polygons. Therefore we calculate regional reference values using regions and bioclimatic zones as stratification. We calculate the mean reference value for each unique combination of region-bioclimatic zone. When calculating the index for each population polygon, the reference value is inherited from the region-bioclimatic zone it falls within.
+
+Once we have the reference vegetation height for a given ecosystem type and region-bioclimatic zone, we need to define the minimum (or worst/bad) condition. We use the 90th percentile of LiDAR-derived vegetation heights within "nasjonalt grunnkart skog" polygons to define a climax vegetation successional stage where gjengroing is at its most extreme, and thus what is the poor ecological condition for naturally open areas. In order not to include forest patches that have recently been harvested, we mask out any forest which has been clear-cut since 1986. Here, 1986 is a hard limit defined by the clear-cut dataset which was based on Landsat imagery. Therefore we can be assured that we are measuring forest that is at least 35 years old. For the natural open areas, however, we set a threshold (0.8 m) for the minimum condition (explained below).
+
+To define the ecological condition of the population polygon, we measure 50th percentile of LiDAR-derived vegetation height. Population polygons should be approximately compatible with the NiN ecosystem types, and we have thus used three different data sources for the three different ecosystem types. For våtmark, we use types in nasjonalt grunnkart defined as Klasse 7 Våtmark. For åpen, we have used the GRUK dataset. For semi-natural, we have used the ASO dataset.
+
+The vegetation height percentiles are then scaled to between 0 and 1 using a sigmoid transformation [Oliver at al. (2021)](https://www.sciencedirect.com/science/article/pii/S1470160X21000066).
```

Comment:
Please add a little more reasoning why so, and why not say an expoential transformation as suggested here: https://github.com/NINAnor/ecosystemCondition/issues/156